<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>choosechar</class>
 <widget class="QDialog" name="choosechar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="char2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>300</y>
     <width>120</width>
     <height>180</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="char3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>300</y>
     <width>120</width>
     <height>180</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="char4">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>300</y>
     <width>120</width>
     <height>180</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>550</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="char1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>120</width>
     <height>180</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
